charts/openshift-metering/templates/hive/hive-server-statefulset.yaml

```diff
@@ -77,15 +78,32 @@ spec:
       - name: copy-starter-hive
         image: "{{ .Values.hive.spec.image.repository }}:{{ .Values.hive.spec.image.tag }}"
         imagePullPolicy: {{ .Values.hive.spec.image.pullPolicy }}
-        command:
-        - '/bin/bash'
-        - '-c'
-        - 'cp -v -L -r -f /hadoop-starting-config/* /hadoop-config/'
+        command: ["/hive-scripts/copy-hadoop-config.sh"]
+{{- if or .Values.hive.spec.config.azure.secretName .Values.hive.spec.config.azure.createSecret }}
+        env:
+        - name: AZURE_STORAGE_ACCOUNT_NAME
+          valueFrom:
+            secretKeyRef:
+              name: "{{ .Values.hive.spec.config.azure.secretName | default "hive-azure-credentials" }}"
+              key: azure-storage-account-name
+        - name: AZURE_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: "{{ .Values.hive.spec.config.azure.secretName | default "hive-azure-credentials" }}"
+              key: azure-secret-access-key
+{{- end }}
         volumeMounts:
         - name: hadoop-config
           mountPath: /hadoop-config
         - name: hadoop-starting-config
           mountPath: /hadoop-starting-config
+        - name: hive-scripts
+          mountPath: /hive-scripts
+      volumes:
```

Comment:
I wonder if we should move this section (`volumes`) to be under `containers:`, which is what we do in the other statefulsets/deployments (`initContainers` -> `containers` -> `volumes`)